content/cluster-autoscaling/cluster-autoscaling.md

```diff
@@ -235,6 +235,10 @@ A complete list of parameters is available on [Github](https://github.com/kubern
 | scale-down-candidates-pool-ratio | A ratio of nodes that are considered as additional non empty candidates for scale down when some candidates from previous iteration are no longer valid. Lower value means better CA responsiveness but possible slower scale down latency. Higher value can affect CA performance with big clusters (hundreds of nodes). Set to 1.0 to turn this heuristics off - CA will take all nodes as additional candidates. | 0.1 |
 | scale-down-candidates-pool-min-count | Minimum number of nodes that are considered as additional non empty candidates for scale down when some candidates from previous iteration are no longer valid. When calculating the pool size for additional candidates we take `max(#nodes * scale-down-candidates-pool-ratio, scale-down-candidates-pool-min-count)` | 50 |
 
+## Talk
+
+Please check 2020 KubeCon EU talk [Autoscaling and Cost Optimization on Kubernetes: From 0 to 100 - Guy Templeton, Skyscanner & Jiaxin Shan, Amazon](https://sched.co/Zemi) for more details.
```

Comment:
How do you feel about "Additional Resources" instead of "Talk"

Reply:
Might be cleaner to do this as a table, too. e.g. https://github.com/kubeflow/kfserving/blob/master/docs/PRESENTATIONS.md

Reply:
Good point. i will make the change